verilog/preprocessor/verilog_preprocess.cc

```diff
@@ -586,6 +586,20 @@ absl::Status VerilogPreprocess::HandleTokenIterator(
   return absl::OkStatus();
 }
 
+// Adds a define passed to the tool with +define+<foo>[=<value>].
```

Comment:
We need to specify lifetime expectations here in the comment: the string-views need to point to valid memory for the lifetime of this `VerilogPreprocess` object (as we retain just the string_views, so they need to be available).

Later, we for error messages, we might want to be able to point where that was defined, but that can be a TODO for later (ideally, I'd like to have FileList also just contain string_views, because then it can be exactly pin-pointed from where things where coming from)